lib/HLSL/DxilLegalizeSampleOffsetPass.cpp

```diff
@@ -198,19 +260,19 @@ void DxilLegalizeSampleOffsetPass::CollectIllegalOffsets(
       if (CI->getParent()->getParent() != &CurF)
         continue;
 
-      CollectIllegalOffset(CI, illegalOffsets);
+      CollectIllegalOffset(CI, opcode, illegalOffsets);
```

Comment:
opcode included to determine offset operand locations. It could be derived from the CI, but it's already available here anyway.